src/react/databrowser/buckets/__tests__/BucketCreate.test.jsx

```diff
@@ -0,0 +1,115 @@
+        expect(component.find('#zk-error-banner').text()).toContain(errorMessage);
+        component.unmount();
+    });
+
+    // TESTING INPUT NAME:
+    // 1) empty name input
+    // 2) name input < 3 characters
+    // 3) name input > 63 characters
+
+    const tests = [
+        {
+            description: 'should render an input form error when submitting with an empty name',
+            testValue: '',
+            expectedEmptyNameError: ' "Name" is not allowed to be empty ',
+            expectedMinLengthNameError: null,
+            expectedMaxLengthNameError: null,
+        },
+        {
+            description: 'should render an input form error when submitting with an name.length < 3',
+            testValue: 'ab',
+            expectedEmptyNameError: null,
+            expectedMinLengthNameError: ' "Name" length must be at least 3 characters long ',
+            expectedMaxLengthNameError: null,
+        },
+        {
+            description: 'should render an input form error when submitting with an name.length > 63',
+            testValue: 'Z4VbHlmEKC0a8n85FEneHN6EhBwFSkmSh4tGOKy53ktdmQlwq5xJVi7hm32jFuKB',
+            expectedEmptyNameError: null,
+            expectedMinLengthNameError: null,
+            expectedMaxLengthNameError: ' "Name" length must be less than or equal to 63 characters long ',
+        },
+    ];
+
+    tests.forEach(t => {
+        it(t.description, async () => {
+            const component = await reduxMountAct(<BucketCreate/>);
+
+            await act(async () => {
+                const input = component.find('input#name');
+                input.getDOMNode().value = t.testValue;
+                input.getDOMNode().dispatchEvent(new Event('input'));
+
+                component.find('Button#create-account-btn').simulate('click');
+            });
+
+            if (t.expectedEmptyNameError !== null) {
+                expect(component.find('ErrorInput#error-name').text()).toContain(t.expectedEmptyNameError);
+            } else if (t.expectedMinLengthNameError !== null) {
+                expect(component.find('ErrorInput#error-name').text()).toContain(t.expectedMinLengthNameError);
+            } else if (t.expectedMaxLengthNameError !== null) {
+                expect(component.find('ErrorInput#error-name').text()).toContain(t.expectedMaxLengthNameError);
+            }
+            component.unmount();
+        });
+    });
+
+    it('Should call clearServerError when clicking on cancel button in the form', async () => {
+        const clearError = jest.spyOn(actions, 'clearError');
+
+        const component = await reduxMountAct(
+            <BucketCreate/>, {
+                uiErrors: {
+                    errorMsg: 'error',
+                    errorType: 'byComponent',
+                },
+            },
+        );
+
+        const cancelButton = component.findWhere(n => n.name() === 'Button' && n.prop('outlined') === true);
```

Comment:
Feel free to add an id: `#cancel-btn` that will help you find the "cancel button" component. 
The component's props might change over time, hence make our tests fail. 
We should avoid relying on them.

Reply:
Ok I'll do that